Items/07_Items.txt

```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
 light for 5m in every direction for about an hour. Burns in rain, mud, even 
 burns underwater. Don't touch the end or you'll take 10 fire damage.
 
+Datapad ($130):
+	A handheld device with a screen and keypad. Used to record
+information up to 1TB. 
+
+Binoculars ($90):
+	A bifocal optical instrument with a lens for each eye, 
+used for viewing distant objects up to 2km. 
+
+Industrial Paint ($35):
+	A metal canister of wear-resistant paint. Can be used 
+with a brush or similar implement to cover up to 20 square meters of
+flooring, wall or other surface. Paint comes in the following colors:
+
+* Black
+* White
+* Red
+* Blue
+* Green
+* Yellow
+* Orange
+* Purple
+* Brown
+
+Adhesive Foam ($45):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in either an area or a thin line. As the solution dries, 
+it expands and hardens to accomodate the space given. Can be used
+to insulate an area, temporarily patch up a damaged wall, 
+seal ducts, etc. One can has 20 cubic meter's worth of foam.
+
+Solvent ($45):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in an area. When this solution comes into contact with
+adhesive foam or similar substances, they will dissolve into a vapor.
+One can has 20 cubic meter's worth of solvent.
```

Comment:
Same as for the adhesive, 20 cu m is probably too much.

Reply:
I mean, the solvent is probably compressed, so it would expand to fill a much larger volume than the can it came out of... but still, 20 m^3 is a _lot_.